internal/cmd/commands/server/server.go

```diff
@@ -645,7 +647,7 @@ func (c *Command) WaitForInterrupt() int {
 		case <-c.SigUSR2Ch:
 			buf := make([]byte, 32*1024*1024)
 			n := runtime.Stack(buf[:], true)
-			c.Logger.Info("goroutine trace", "stack", string(buf[:n]))
+			event.WriteSysEvent(context.TODO(), op, map[string]interface{}{"msg": "goroutine trace", "stack": string(buf[:n])})
```

Comment:
I am seeing several of spots like this that are still using `map[string]interface{}` instead of `event.I`, `event.H`, or `event.D`.

Reply:
ty!